genologics/entities.py

```diff
@@ -765,24 +755,30 @@ class Entity(object):
 
     _TAG = None
     _URI = None
+    _PREFIX=None
 
-    def __new__(cls, lims, uri=None, id=None):
+    def __new__(cls, lims, uri=None, id=None, create_new=None):
```

Comment:
That's  a good point. 
I first though that could be a dict of kwargs but that turned out not work.
I'll change it